src/api/apps/articles/model/save.coffee

```diff
@@ -187,8 +190,15 @@ sanitize = (article) ->
   sanitized
 
 sanitizeLink = (urlString) ->
+  return unless urlString
   u = url.parse urlString
-  if u.protocol then urlString else 'http://' + u.href
+  if u.protocol
+    if u.protocol is "http:"
+      urlString.replace("http:", "https:")
+    else
+      urlString
+  else
+    'https://' + u.href
```

Comment:
I can't tell from this PR where else this method is called. I think we can _only_ transform the protocol for `artsy.net` links. Otherwise, we might be prescribing SSL for an external site that doesn't support it (leading to a broken link).

Reply:
Cool I misunderstood- will update this to force https only to artsy.net

Reply:
This is now updated to only force https for artsy links or links without a protocol. To your question of where this is run, `sanitize` calls the message on the article body text and on explicitly defined article keys. This PR adds a couple new fields to ensure we hit anywhere that site link would be saved.